code/camera/cameradisplay.cpp

```diff
@@ -117,6 +127,28 @@ void CameraDisplay::update_zoom(int value) {
     Q_EMIT zoom_changed(zoom_factor);
 }
 
+void CameraDisplay::gridSelectChanged(int weight_index) {
+        weightSelected = m_ui->weight_list->currentText();
```

Comment:
Some tab formatting issues here. Preprocessor lines (starting with `#`) should not be indented